pkg/cmd/regeneratecerts/regenerate_certificates.go

```diff
@@ -340,7 +340,7 @@ func (o *Options) Run() error {
 		}
 
 		dir := filepath.Join(def.toplevelDir, def.objectType, def.name)
-		err = os.MkdirAll(dir, 0755)
+		err = os.MkdirAll(dir, 0750)
```

Comment:
how can static analysis confirm that this makes sense? Maybe it even does, am just curious.

Reply:
static code analysis will issue false positives. This might or might not make sense; however, in many cases the intent is not to make these files readable by everyone. So, this is a good default to have unless you really want to share this directory.

Reply:
@tnozicka can judge here whether it is good.

Reply:
any more feedback?

Reply:
there are configmaps (public CAs in this dir) so it makes sense for the dir itself to be public

Reply:
sure, I can change these. Also, if we think the rule doesn't make sense we can exclude it entirely.